src/lib.rs

```diff
@@ -0,0 +1,23 @@
+#![warn(
```

Comment:
These are some additional lints I found that are pretty good to have. You can use whatever you want, of course.